models/public/yolo-v4-tiny-tf/yolo-v4-tiny-tf.md

```diff
@@ -0,0 +1,134 @@
+# yolo-v4-tiny-tf
+
+## Use Case and High-Level Description
+
+YOLO v4 is a real-time object detection model based on ["YOLOv4: Optimal Speed and Accuracy of Object Detection"](https://arxiv.org/abs/2004.10934) paper. It was implemented in Keras\* framework and converted to TensorFlow\* framework. For details see [repository](https://github.com/david8862/keras-YOLOv3-model-set). This model was pretrained on COCO\* dataset with 80 classes.
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 6.9289        |
+| MParams           | 6.0535        |
+| Source framework  | Keras\*       |
+
+## Accuracy
+
+Accuracy metrics obtained on COCO\* validation dataset for converted model.
+
+| Metric | Value |
+| ------ | ------|
+| mAP    |     % |
+| [COCO\* mAP (0.5)](http://cocodataset.org/#detection-eval) | % |
+| [COCO\* mAP (0.5:0.05:0.95)](http://cocodataset.org/#detection-eval) | % |
+
+## Input
+
+### Original model
+
+Image, name - `input_1`, shape - `1,416,416,3`, format is `B,H,W,C` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+Scale value - 255.
+
+### Converted model
+
+Image, name - `input_1`, shape - `1,3,416,416`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+
+1. The array of detection summary info, name - `conv2d_20/BiasAdd`, shape - `1,38,38,255`. The anchor values are `23,27, 37,58, 81,82`.
+
+2. The array of detection summary info, name - `conv2d_17/BiasAdd`, shape - `1,19,19,255`. The anchor values are `81,82, 135,169, 344,319`.
```

Comment:
Are you sure about shape?

Reply:
Forgot to fix it, thanks! 38, 19 is for 608x608.